README.md

```diff
@@ -39,14 +37,45 @@ To run:
 
 
 ### analysis.py
-Script that can be used to:
-1. print total snf amount and isotope mass (analysis.snf(cursor))
-2. create stacked bar chart of power capacity and number of reactors per region (analysis.plot_power(cursor))
-3. create a multi-line plot chart of waste isotope per time (analysis.isotope_vs_time(cursor))
 
-The following command on the terminal will execute all three:
+Input : CYCLUS output file (.sqlite)
+`python analysis.py [outputfile]`
+
+Most functions return a dictionary of lists (timeseries of a value)
+that can be used to plot a stacked bar chart or a line plot.
+
+Major analysis functions include:
```

Comment:
Great. A few tests are better than none. 

Re: sqlite files. You can indeed. See cyclus for examples of how we do it there. That said, it's simpler if your functions are smaller. That is, many of these functions could be split up between two steps (1: constructing the 
 sqlite queries, and 2: executing those queries).